README.md

```diff
@@ -15,6 +15,19 @@ Following are some useful traits and practices for Developers. This is by no mea
 * Be responsible. Deliver on your promises, set realistic expectations and communicate when you cannot meet your goals.
 * Be flexible. Process is good, but progress is better. Sometimes you have to work around process to make progress.
 
+## Responsibilities of a Junior Developer
+As a junior, your primary role is to learn the domain knowledge of your workplace as well as tricks of the trade from your fellow developers. This is a list of recommendations similar to that of the developer, but with a focus on how it applies to you.
+
+* Be a team player. Follow your team's conventions and processes. However, as a junior, you should ask questions. If you don't know something, you should be asking.
```

Comment:
A bold font may read better for the TL;DR folks.
ie:
* **Be a team player:** Follow your team's...